Dockerfile

```
@@ -0,0 +1,18 @@
+FROM rust:1.56 AS builder
+
+WORKDIR /app
+
+COPY . ./
+
+RUN cargo build --release
+
+FROM ubuntu AS final
```

Is this label necessary if we don't reference it later?

```suggestion
FROM ubuntu
```

---

Fixed.